pkg/controller/migplan/migplan_controller.go

```diff
@@ -71,6 +73,32 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Watch for changes to deployment registry referenced by MigPlan that have running migrations
+	err = c.Watch(
+		&registryHealth{
+			hostClient: mgr.GetClient(),
+			planLabels: map[string]string{
+				migapi.MigplanMigrationRunning: "true",
+			},
+			Interval: time.Second * 5},
```

Comment:
I will update the probe with 3-second timeout and 15 seconds of initial delay. I would still keep this at 5 second, since it will only mean at worst case scenario we will have 2 unnecessary checks. Can discuss this further on a call if this does not sound okay

Reply:
I'm not sure I understand the need to react so quickly to registry problems, seems like waiting ~30s for the issue to be surfaced by the controller would be acceptable and have less APIserver strain, but I may not have some key piece of info still.